.gitignore

```diff
@@ -395,4 +395,7 @@ FodyWeavers.xsd
 *.msp
 
 # JetBrains Rider
-*.sln.iml
\ No newline at end of file
+*.sln.iml
+
+# Firebase
+FirebaseServiceAccountKey.json
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-easyspeak/pull/44/files
We have this file in this PR, so no need to create conflicts

Reply:
I thought it was merged before